styleguide/_themes/derek/scss/components/_footer.scss

```diff
@@ -1,3 +1,87 @@
 .footer{
-  
-}
\ No newline at end of file
+  ul{
+    list-style: none;
+    padding: 0;
+    margin: 0;
+  }
+  .footer-menu {
```

Comment:
You should be able to pull `.footer-menu{}` to the top level instead of nested inside `.footer`